.github/workflows/otp_pkg_prerelease.yaml

```diff
@@ -0,0 +1,171 @@
+  upload_artifacts:
+    runs-on: ubuntu-latest
+    needs: [define-matrix, otp-gen-matrix]
+    strategy:
+      matrix:
+        otp: ["${{ fromJson(needs.define-matrix.outputs.otp-version) }}"]
+        deps: ['grisp', 'grisp, grisp_cryptoauth']
+        rebar3: ['3.24.0']
+    outputs:
+      artifacts: ${{ steps.matrix-def.outputs.otp-version }}
+      upload-output: ${{ steps.set-upload-output.outputs.upload-output }}
+    steps:
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{matrix.otp}}
+          rebar3-version: ${{ matrix.rebar3 }}
+      - name: Install GRiSP Plugin
+        run: |
+          mkdir -p ${HOME}/.config/rebar3/
+          echo "{plugins, [rebar3_hex,rebar3_grisp]}." > ${HOME}/.config/rebar3/rebar.config
+          rebar3
+      - name: Generate Dummy Project
+        run: |
+          rebar3 grisp configure -i false --otp_version="=${{matrix.otp}}" --dest="_deploy"
+          mkdir robot/_deploy
+          sed -i 's/grisp/${{matrix.deps}}/g' robot/src/robot.app.src
+          sed -i '/{deps, \[/,/\]}.*/{
+          N
+          N
+          s/{deps, \[\n[[:space:]]*grisp\n\]}.*/{deps, [${{matrix.deps}}]}./
+          }' robot/rebar.config
+          cat robot/rebar.config
+      - name: Build Package
+        id: deploy
+        working-directory: robot
+        run: |
+          sed -i '/{grisp, \[/a\
+          '"{build, []}," rebar.config
+          rebar3 grisp report
+          PKG_HASH=$(grep -o '<<"[0-9a-f]\{64\}">>' _grisp/report/hash.txt | tail -n 1 | sed 's/[<">]//g')
+          PKG_NAME=grisp_otp_build_${{matrix.otp}}_${PKG_HASH}.tar.gz
+          echo "pkg_name=$PKG_NAME" >> $GITHUB_ENV
+      - name: Download Package Artifact
+        id: artifact-download
+        uses: actions/download-artifact@v4
+        continue-on-error: true
+        with:
+          name: ${{env.pkg_name}}
+      - name: Download from S3 if artifact not found
+        id: download-s3
+        if: steps.artifact-download.outcome == 'failure'
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.GRISP_S3_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.GRISP_S3_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: "us-east-1"
+        run: |
+          aws s3 cp s3://grisp/platforms/grisp2/otp/${{env.pkg_name}} .
+      - name: Upload to S3
+        id: upload-s3
+        if: ${{ steps.artifact-download.outcome == 'success' }}
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.GRISP_S3_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.GRISP_S3_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: "us-east-1"
+        run: |
+          aws s3 cp --acl public-read --storage-class INTELLIGENT_TIERING \
+          ${{env.pkg_name}} \
+          s3://grisp/platforms/grisp2/otp/
+      - name: Set S3 upload output
+        if: ${{ steps.upload-s3.outcome == 'success' }}
+        run: echo "upload-output=true" >> $GITHUB_OUTPUT
+        id: set-upload-output
+
```

Comment:
@Fantarina I see this version does not trigger the slack message.
We would actually like to broadcast release candidates too. 
So I think that using the original workflow would be ok.